nuxt.config.js

```diff
@@ -284,6 +284,10 @@ export default defineNuxtConfig({
     // },
   },
 
+  routeRules: {
+    '/ahp/collection/13': { sitemap: { changefreq: 'daily', priority: 0.3 } },
+  },
+
```

Comment:
It could also work.

![image](https://github.com/kodadot/nft-gallery/assets/31397967/7e435af5-ad40-4354-9489-67aa21a1b627)

Reply:
Hmmmmm nice I'll look Into that